packages/pagination/PaginationControls.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+import { usePagination } from './Pagination';
+
+const PaginationControls = ({ directionLinks = false, autoHide = true }) => {
+  const { pages, page, setPage } = usePagination();
+
+  const { number: current } = page;
+
+  return (
+    <React.Fragment>
+      {pages.length > 1 || !autoHide ? (
+        <Pagination>
+          {directionLinks ? (
+            <PaginationItem
+              disabled={current === 1}
+              onClick={() => (current === 1 ? null : setPage(current - 1))}
+            >
+              <PaginationLink previous />
+            </PaginationItem>
+          ) : (
+            ''
+          )}
+          {pages.map(pageNumber => (
+            <PaginationItem
+              key={pageNumber}
+              active={current === pageNumber}
+              onClick={() => setPage(pageNumber)}
+            >
+              <PaginationLink>{pageNumber}</PaginationLink>
+            </PaginationItem>
+          ))}
+          {directionLinks ? (
+            <PaginationItem
+              disabled={current === pages.length}
+              onClick={() =>
```

Comment:
`PaginationItem` output's an `li` which is a non-interactive element, you probably want to change the `tag` or move the `onClick` to the `PaginationLink` which will output a `button` (and, if you do that, also add `type="button"` to it so that it doesn't trigger submits if it appears within a form).